pkg/controller/cloud/node_lifecycle_controller.go

```diff
@@ -139,7 +138,7 @@ func (c *CloudNodeLifecycleController) MonitorNodes() {
 
 		if status == v1.ConditionTrue {
 			// if taint exist remove taint
-			err = controller.RemoveTaintOffNode(c.kubeClient, node.Name, node, ShutdownTaint)
+			err = cloudnodeutil.RemoveTaintOffNode(c.kubeClient, node.Name, node, ShutdownTaint)
```

Comment:
Curious where else `RemoveTaintOffNode` and `AddOrUpdateTaintOnNode` are referenced from `pkg/controller`

Reply:
for `RemoveTaintOffNode` the other in-tree refs are:
```
pkg/controller/util/node
pkg/controller/nodelifecycle/
test/e2e
```

for `AddOrUpdateTaintOnNode ` the in-tree refs are:
```
pkg/controller/util/node
test/e2e
```